### PR TITLE
Add staking contract to deployment

### LINKED ISSUE #170: Test staking contract full stack
We now have a stable v0.8.0-alpha3 release and integration tests for the staking contract in `wasmd`.

To fully demo this, we need some very high level tests via cosmwasm-js showing the contract in use. It only needs to cover a happy path, but mainly showing client interaction down to the contract working.

### PR DESCRIPTION
**Pull request by webmaster128**
~Based on #176~

Closes #170

### CODE REVIEW

File: packages/demo-staking/src/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    {
+      const query: QueryMsg = { balance: { address: userAddress } };
+      const response: BalanceResponse = await ownerClient.queryContractSmart(contractAddress, query);
+      expect(response).toEqual({
+        balance: "112233",
+      });
+    }
+
+    const unbondMsg: HandleMsg = { unbond: { amount: "112233" } };
+    // await ownerClient.execute(contractAddress, unbondMsg);
```
Comment (webmaster128) [inline packages/demo-staking/src/index.spec.ts:126]: @ethanfrey uncommenting this line, i.e. unbonding from an account that did not bond leads to some fun panic:

```
Error when posting tx B68E0BCC2A8A3018E43A9AA1E4A83DC4527286820E4EE5CFD8B95A561C4540EC. Code: 4; Raw log: 
  github.com/CosmWasm/wasmd/x/wasm/internal/keeper.Keeper.Execute
        github.com/CosmWasm/wasmd@/x/wasm/internal/keeper/keeper.go:198
  github.com/CosmWasm/wasmd/x/wasm.handleExecute
        github.com/CosmWasm/wasmd@/x/wasm/handler.go:104
  github.com/CosmWasm/wasmd/x/wasm.NewHandler.func1
        github.com/CosmWasm/wasmd@/x/wasm/handler.go:35
  github.com/cosmos/cosmos-sdk/baseapp.(*BaseApp).runMsgs
        github.com/cosmos/cosmos-sdk@v0.38.3/baseapp/baseapp.go:669
  github.com/cosmos/cosmos-sdk/baseapp.(*BaseApp).runTx
        github.com/cosmos/cosmos-sdk@v0.38.3/baseapp/baseapp.go:638
  github.com/cosmos/cosmos-sdk/baseapp.(*BaseApp).DeliverTx
        github.com/cosmos/cosmos-sdk@v0.38.3/baseapp/abci.go:207
  github.com/tendermint/tendermint/abci/client.(*localClient).DeliverTxAsync
        github.com/tendermint/tendermint@v0.33.3/abci/client/local_client.go:88
  github.com/tendermint/tendermint/proxy.(*appConnConsensus).DeliverTxAsync
        github.com/tendermint/tendermint@v0.33.3/proxy/app_conn.go:73
  github.com/tendermint/tendermint/state.execBlockOnProxyApp
        github.com/tendermint/tendermint@v0.33.3/state/execution.go:293
  github.com/tendermint/tendermint/state.(*BlockExecutor).ApplyBlock
        github.com/tendermint/tendermint@v0.33.3/state/execution.go:131
  github.com/tendermint/tendermint/consensus.(*State).finalizeCommit
        github.com/tendermint/tendermint@v0.33.3/consensus/state.go:1431
  github.com/tendermint/tendermint/consensus.(*State).tryFinalizeCommit
        github.com/tendermint/tendermint@v0.33.3/consensus/state.go:1350
  github.com/tendermint/tendermint/consensus.(*State).enterCommit.func1
        github.com/tendermint/tendermint@v0.33.3/consensus/state.go:1285
  github.com/tendermint/tendermint/consensus.(*State).enterCommit
        github.com/tendermint/tendermint@v0.33.3/consensus/state.go:1322
  github.com/tendermint/tendermint/consensus.(*State).addVote
        github.com/tendermint/tendermint@v0.33.3/consensus/state.go:1822
  github.com/tendermint/tendermint/consensus.(*State).tryAddVote
        github.com/tendermint/tendermint@v0.33.3/consensus/state.go:1642
  github.com/tendermint/tendermint/consensus.(*State).handleMsg
        github.com/tendermint/tendermint@v0.33.3/consensus/state.go:709
  github.com/tendermint/tendermint/consensus.(*State).receiveRoutine
        github.com/tendermint/tendermint@v0.33.3/consensus/state.go:660
  execute wasm contract failed: %!v(PANIC=Error method: unknown error variant): failed to execute message; message index: 0
```

Comment (ethanfrey) [reply packages/demo-staking/src/index.spec.ts:136]: Huh, definitely looks like a bug, maybe in the staking contract logic, maybe in the go side.

Let me fix this before 0.8

Comment (webmaster128) [reply packages/demo-staking/src/index.spec.ts:136]: Fixed in https://github.com/CosmWasm/go-cosmwasm/pull/89

**Comment by webmaster128** [comment]
@ethanfrey this is ready for review/merge

**Pull request merged**